README.md

```diff
@@ -37,5 +37,22 @@ using StringThings.Extensions;
 "ABC".ContainsIgnoreCase("abc");
 ```
 
+
+## Take First/Last Characters Extensions
+No more length errors when using substring to get the leading or trailing characters from a string. Let this extension do the work for you of verifying that your substring doesn't exceed the length of the string you are evaluating.
```

Comment:
I really appreciate you matching my energy with this text 😂

Reply:
"And YOU get a string, and YOU get a string!"